queries.sql

```diff
@@ -1,11 +1,67 @@
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals WHERE name LIKE '%mon%';
-SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
 SELECT * FROM animals WHERE neutered = 't' AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT * FROM animals
-SELECT * FROM animals WHERE neutered = 't';
\ No newline at end of file
+SELECT * FROM animals WHERE neutered = 't';
+
+/* Rollback practise */
+ BEGIN;
+ UPDATE animals SET species = 'unspecified';
+ ROLLBACK;
+
```

Comment:
- [ ] To verify that the changes were undone please add this line 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/215282683-aa2ce3c5-f735-4142-bdd6-4a54425ffb16.png)